foundations/javascript_basics/revisiting_rock_paper_scissors.md

```diff
@@ -53,8 +53,9 @@ For example: if you have a bug in a new feature you’re working on that you can
    3. Add a `div` for displaying results and change all of your `console.log`s into DOM methods.  
    4. Display the running score, and announce a winner of the game once one player reaches 5 points.  
    5. You will likely have to refactor (rework/rewrite) your original code to make it work for this. That's OK! Reworking old code is an important part of a programmer's life.  
-   6. Once you're all done with your UI and make sure everything's satisfactory, it's time to merge our current branch `rps-ui` to our main branch.  
-3. Now let's take a look at how we can merge these changes back to our `main` branch.  
+   6. Once you're all done with your UI and made sure everything's satisfactory, commit your changes to the `rps-ui` branch.
+   7. Now it's time to merge it to our `main` branch.
+3. Let's take a look at how we can merge these changes back to our `main` branch.  
```

Comment:
I think we can actually do away the with verbiage in your step 7/that is currently in the lesson already about merging, just because the step that follows already says "Now let’s take a look at how we can merge these changes...". It's kind of redundant verbiage. If you can make the following changes, I think this will be good to merge:

1. Remove the line "7. Now it's time to merge it to our `main` branch." completely
2. Edit the line that follows to this:

```suggestion
3. Now let's take a look at how we can merge the changes from our `rps-ui` branch back to our `main` branch.  
```

Reply:
Yes, I think that is appropriate, and I have made the changes.